MonoGame.Framework/BoundingFrustum.cs

```diff
@@ -123,7 +123,7 @@ internal string DebugDisplayString
         #region Constructors
 
         /// <summary>
-        /// Creates a new instance of <see cref="BoundingFrustum"/> class.
+        /// Constructs a bounding frustum.

```

Comment:
> lets write same constuctors as Matrix to all documented classes for unification

Right...  we have to remember the primary goal is to communicate how things work.